tensorrt_llm/_torch/auto_deploy/transform/library/quantization.py

```
@@ -22,6 +22,7 @@
     should_skip_quantization,
 )
 from ..interface import BaseTransform, TransformInfo, TransformRegistry
+from tensorrt_llm.quantization.quant_annotation import QuantAnnotation, annotate_node, has_quant_annotation, get_quant_annotation
```

Ruff (0.12.2)

Line too long (129 > 120)

(E501)